odoo/models.py

```diff
@@ -5521,6 +5521,33 @@ def onchange(self, values, field_name, field_onchange):
 
         return result
 
+    #
+    # Cleanup
+    #
+
+    @api.model
+    def _expunge_bad_records(self):
```

Comment:
I personally don't like much the method name.
What about a simpler name like `_cleanup_model` or `_cleanup_table` ?